test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/ModelDirectiveTest.cs

```diff
@@ -259,6 +259,13 @@ private DocumentIntermediateNode CreateIRDocument(RazorEngine engine, RazorCodeD
                 }
             }
 
+            // InheritsDirectivePass needs to run before ModelDirective.
+            var pass = new InheritsDirectivePass()
```

Comment:
These unit tests were verifying the right code. Without the `InheritsDirectivePass` running, the class won't have a basetype set. Removing the unnecessary code in ModelDirective uncovered this problem.